phobos-perf/ansible.cfg

```diff
@@ -0,0 +1,16 @@
+[defaults]
```

Comment:
Can we use the existing ansible.cfg (perhaps symlink it in place) and use the bootstrap-ansible.sh to ensure the correct roles are cloned?